.github/workflows/test_suite.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest, macos-13]
```

Comment:
This change should be temporary -- `fftvis` tests fail with the latest M1/M2 chips, but we don't know why yet. This restricts tests to the older intel chips. I'll make an issue and @tyler-a-cox will investigate, but I don't think it should hold us up.

Reply:
I'll take a look at this soon. Thanks for making an issue!